tutorials/quickstart.eve.md

```diff
@@ -0,0 +1,90 @@
+# Quickstart
+
+In this document we're going to give a 5 minute introduction to concepts essential to Eve. If you've never used Eve before, you're in the right place.
+
+## Adding records to Eve
+
+Eve represents data as records, key value pairs attached to a unique ID. Eve programs are made up of small composable blocks that search for and create records. Let's start with a block that adds a record to Eve:
+
+```eve
+commit
+  [#greeting text: "hello world"]
+```
+
+This record is tagged `#greeting`, and has an attribute "text" with the value "hello world". Although tags are attributes like any other,they have special syntax because they are used often.
+
+## Finding records in Eve
+
+We can search for records, and then bind new records to those that are found. Eve finds every record matching the supplied patterns, and then binds records dependent on them. If no records match the search, then the block does not run at all. A block will only run if every record in a search matches at least one result. Let's search for the `#greeting` we just committed, and then display it in a div:
```

Comment:
"every record in a search" -> "every pattern in a search"? De-emphasizing patterns makes sense where possible, but I think it's *probably* better to use separate verbiage here? Since otherwise you're writing a record to match a record and it becomes a bit tough to disambiguate between the concrete thing and the template.